python/semantic_kernel/connectors/ai/ai_exception.py

```diff
@@ -28,6 +28,8 @@ class ErrorCodes(Enum):
         InvalidConfiguration = 8
         # The function is not supported.
         FunctionTypeNotSupported = 9
+        # The HF task is not supported.
+        UnsupportedHFTask = 10
```

Comment:
could we avoid branded error codes, maybe reuse an existing code or introduce one that can be reused? you can use error messages to provide further details

Reply:
reusing ServiceError instead